bin/GameIndex.dbf

```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
 Name   = Homura [Taito the Best]
 Region = NTSC-J
 ---------------------------------------------
+Serial = SLKA-25080
+Name   = .hack//Kansen Kakudai Vol. 1
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25138
+Name   = .hack//Akushou Heni Vol. 2
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25145
+Name   = .hack//Shinshoku Osen Vol. 3
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25174
+Name   = .hack//Zettai Houi Vol. 4
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-15023
```

Comment:
This one is not in order, should be early in the list. All of the games are in order by serial number and letter prefixes.